automation/test.sh

```diff
@@ -11,7 +11,13 @@ sbt clean
 
 cat testout.txt | sed -n -e '/Run completed/,$p' | sed $'s,\x1b\\[[0-9;]*[a-zA-Z],,g' > output/testsummary.txt
 
-curl -F file=@output/testsummary.txt -F channels=#${TEST_CHANNEL} -F token=${SLACK_API_TOKEN} -F filename=fiab-test-${BUILD_NUMBER} https://slack.com/api/files.upload
+if [[ $TEST_EXIT_CODE != 0 ]]; then COLOR=danger; else COLOR=good; fi
+BUILD_NAME=fiab-test-${BUILD_NUMBER}
+
+ATTACHMENTS='[{"color":"'"${COLOR}"'","text":"'"${BUILD_NAME}"'"]'
+
+curl -F channels=#${SLACK_CHANNEL} -F token=${SLACK_API_TOKEN} -F attachments=${ATTACHMENTS} https://slack.com/api/chat.postMessage
```

Comment:
does this work without inputs to `channels` ?

Reply:
(since it is commented out)

Reply:
it only looks commented out; it's actually  `#` character in the option